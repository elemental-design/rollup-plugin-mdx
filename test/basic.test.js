import React from 'react';

import renderer from 'react-test-renderer';

import { BasicContainer } from './dist/basic.es';

// require('source-map-support').install();

// process.chdir(__dirname);

test('Link changes the class when hovered', async () => {
  

  const component = renderer.create(
    <div>
      <BasicContainer />
    </div>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

/*
const bundle = await rollup
    .rollup({
      input: 'samples/basic.js',
      plugins: [
        mdx(),
        babel({
          exclude: 'node_modules/**',
        }),
      ]
    });

  const { output: [{ code }] } = await bundle.generate({
    format: 'es'
  });
  const exports = {};
  const fn = new Function('exports', code);

  fn(exports);
  const { BasicContainer } = exports;
*/
