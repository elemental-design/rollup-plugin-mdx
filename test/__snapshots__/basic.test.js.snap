// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div>
  <div>
    <h1>
      Test Title
    </h1>
    <p>
      Test document with 
      <strong>
        bold
      </strong>
       and 
      <em>
        italics
      </em>
       text.
    </p>
  </div>
</div>
`;
